daemon/src/model/cfd.rs

```diff
@@ -313,6 +294,36 @@ fn calculate_profit(
     Ok((Amount::ZERO, Usd::ZERO))
 }
 
+/// Calculates the buyer's margin in BTC
+///
+/// The margin is the initial margin and represents the collateral that both parties have to come up with to satisfy the contract.
```

Comment:
Isn't it only the buyer who has to come up with this margin?

Reply:
yeah, sorry for the confusion. 
It should probably not be called `margin` for the seller at the moment, because the seller cannot margin trade at the moment. For the maker (being the seller) it is always just the BTC amount of the quantity given the price atm. Weirdly, the leverage is also set to `5` for the seller at the moment.

Would it be more correct to also apply the leverage for the seller? (we don't have that in our spreadsheet, but I think it would be more correct)

I still used the same terminology as for the taker because at some point I would expect the maker to be able to use a margin (through the leverage) as well. But I am not sure that makes sense.

Reply:
I was actually only referring to the sentence: 

```
/// Calculates the buyer's margin in BTC
///
/// The margin is the initial margin and represents the collateral that both parties have to come up with to satisfy the contract.
```

It says first: `calculate the buyer's margin` and then further down: `.. that both parties have to come up with`. 

Imho everything in this function is correct, the only thing which I'd change is this: 

```suggestion
/// The margin is the initial margin and represents the collateral that the buyer has to come up with to satisfy the contract.
```


--------


Regarding the topic of allowing both sides to leverage their position: I'm still of the opinion that this does not make sense....